sdk/storage/azure-storage-blob-stress/scenarios-matrix.yaml

```diff
@@ -1,13 +1,13 @@
 matrix:
   scenarios:
-#   this test downloads 1024 bytes, small file, no chunking
+    #   this test downloads 1024 bytes, small file, no chunking
```

Comment:
Would you prefer to pull out some common fields here, or to keep each test case spec independent? If the former I can help with that.

Reply:
common fields would be cool!

Is there a way to reuse the same approach as we have with other test matrices? Like if  I defined one for job generation, I'd do

{
  "resourceProfiles" :  {
      "small": { "cores": 0.5, "memory": "500Mi" },
      "large": { "cores": 2, "memory": "2Gi" },
    },
  "matrix": [
  {
    "resource": "small"
    "sync": [ true, false ],
    "faults":  [ true, false ],
    "blobSize": 1024
    "scenarios": ["downloadcontent", "downloadstream", ...]
  },
  {
    "resource": "large"
    "sync": [ true, false ],
    "faults":  [ true, false ],
    "blobSize": "52428800"
    "scenarios": ["downloadcontent", "downloadstream", ...]
  }
]}

 Could I define something like this in the yaml and do non-sparse generation?